src/Goerli-DssSpell.sol

```diff
@@ -31,31 +42,197 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
     string public constant override description = "Goerli Spell";
 
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant WAD     = 10 ** 18;
+    uint256 constant RAY     = 10 ** 27;
+
+    // address constant CES_WALLET  = 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da;
+    // address constant RISK_WALLET = 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c;
+    address constant DEPLOYER_WALLET = 0xdA0C0de01d90A5933692Edf03c7cE946C7c50445;
+
+    address constant MCD_VEST_MKR_TREASURY = 0xd1B8dFF41F3268fAC524869f4C7dA27232044916;
+
+    uint256 constant APR_01_2021 = 1617235200;
+
+    uint256 constant CURRENT_BAT_MAT          = 150 * RAY / 100;
+    uint256 constant CURRENT_LRC_MAT          = 175 * RAY / 100;
+    uint256 constant CURRENT_ZRX_MAT          = 175 * RAY / 100;
+    uint256 constant CURRENT_UNIV2AAVEETH_MAT = 165 * RAY / 100;
+    uint256 constant CURRENT_UNIV2LINKETH_MAT = 165 * RAY / 100;
+
+    uint256 constant TARGET_BAT_MAT          = 800 * RAY / 100;
+    uint256 constant TARGET_LRC_MAT          = 2600 * RAY / 100;
+    uint256 constant TARGET_ZRX_MAT          = 900 * RAY / 100;
+    uint256 constant TARGET_UNIV2AAVEETH_MAT = 400 * RAY / 100;
+    uint256 constant TARGET_UNIV2LINKETH_MAT = 300 * RAY / 100;
+
+    function add(uint x, uint y) internal pure returns (uint z) {
```

Comment:
Minor nit, but we might want to prefix the internal functions `_add()` so they don't conflict with DS-Math and they are easier to identify in code.